### PR TITLE
Fix/coinatmradar fix 7.5 - error 400 request response fix

### PR DESCRIPTION
**Pull request by hellypi**

### CODE REVIEW

**Comment by hellypi** [comment]
['Status code 400' errors related to Coin ATM Radar on v7.5](https://trello.com/c/LcDb19y6/404-status-code-400-errors-related-to-coin-atm-radar-on-v75)

File: lib/plugins.js [hunk 1]
```diff
@@ -58,7 +58,6 @@ function plugins (settings, deviceId) {
         cashOut: cashOutCommission && rate.bid.div(cashOutCommission).round(5)
       }
     })
-
```
Comment (RafaelTaranto) [inline lib/plugins.js]: revert this change

File: lib/coinatmradar/coinatmradar.js [hunk 2]
```diff
@@ -139,11 +126,10 @@ function sendRadar (data) {
     timeout: TIMEOUT,
     maxContentLength: MAX_CONTENT_LENGTH
   }
+  // console.log('%j', data)
```
Comment (RafaelTaranto) [inline lib/coinatmradar/coinatmradar.js]: Remove the console.log

File: lib/coinatmradar/test/coinatmradar.test.js [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+          })
+        )
+      })
+    )
+  })
+  return schema.validate(data)
+}
+
+const db = require('../../db')
+jest.mock('../../db')
```
Comment (RafaelTaranto) [inline lib/coinatmradar/test/coinatmradar.test.js]: requires should be grouped at the top of the file

File: lib/coinatmradar/coinatmradar.js [hunk 4]
```diff
@@ -139,11 +126,10 @@ function sendRadar (data) {
...
 
-  console.log('%j', data)
-
-  return axios(config)
-    .then(r => console.log(r.status))
+  return axios.default(config)
+  .then(r => console.log(r.status))
```
Comment (RafaelTaranto) [inline lib/coinatmradar/coinatmradar.js]: this is not properly formatted, the .then should be on the next indent level

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Comment by RafaelTaranto** [comment]
@joshmh can we squash and merge this one?

**Pull request merged**